client/src/components/favorites-sidebar.tsx

```
@@ -0,0 +1,168 @@
+
+  const handleCopyUrl = async (url: string, id: number) => {
+    try {
+      await navigator.clipboard.writeText(url);
+      setCopiedId(id);
+      toast({
+        title: t('result.copiedTitle'),
+        description: t('result.copiedDescription'),
+        variant: 'success',
+      });
+      setTimeout(() => setCopiedId(null), 2000);
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)